dashboard/app/views/home/_privacy.md

```diff
@@ -187,35 +189,43 @@ Whenever we share Student personal data, we hold our partners to privacy and sec
 
 In particular, we do not share any Personal Information you provide with our donors or sponsors (other than the "Public Reporting" of de-identified reports as described below).
 
+### We may share Student data with your Teacher and limited Teacher information with the Student
```

Comment:
Should it be `share Student data with the Student's Teacher`? Unclear who 'your' is

Reply:
Actually in general, I'm not sure if it matters. We often use you/your and switch between the context of student and teacher. 🤷‍♀️